istio/istio.go

```diff
@@ -424,6 +446,37 @@ func (iClient *IstioClient) ApplyOperation(ctx context.Context, arReq *meshes.Ap
 		return &meshes.ApplyRuleResponse{
 			OperationId: arReq.OperationId,
 		}, nil
+	case installHipsterShopCommand:
+        go func() {
+            opName1 := "deploying"
+            if arReq.DeleteOp {
+                opName1 = "removing"
+            }
+            if err := iClient.executeHipsterShopInstall(ctx, arReq); err != nil {
+                iClient.eventChan <- &meshes.EventsResponse{
+                    OperationId: arReq.OperationId,
+                    EventType:   meshes.EventType_ERROR,
+                    Summary:     fmt.Sprintf("Error while %s the Hipster Shop", opName1),
+                    Details:     err.Error(),
+                }
+                return
+            }
+            opName := "deployed"
+            if arReq.DeleteOp {
+                opName = "removed"
+            }
+            iClient.eventChan <- &meshes.EventsResponse{
+                OperationId: arReq.OperationId,
+                EventType:   meshes.EventType_INFO,
+                Summary:     fmt.Sprintf("Book Info app %s successfully", opName),
+                Details:     fmt.Sprintf("The Hipster Shop is now %s.", opName),
```

Comment:
Same here.
It will nice if you can use the same name in all the places for consistency.